README.md

```diff
@@ -115,4 +115,26 @@ And inside the running container shell execute:
 ros2 run teleop_twist_keyboard teleop_twist_keyboard
 ```
 
+## Quick Start (Webots simulation)
 
+> **Prerequisites**
+>
+> The `compose.sim.webots.yaml` file uses NVIDIA Container Runtime. Make sure you have NVIDIA GPU and the [NVIDIA Container Toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html) installed.
```

Comment:
Is it possible to run webots without nvidia runtime? For ignition-gazebo simulation we have two configs (the one without nvidia runtime is really really slow though).

Reply:
Added in readme running without nvidia.